{"pytest_version": "7.4.3", "$report_type": "SessionStart"}
{"nodeid": "tests57_66_4.py::test_matrix_of_several_numbers", "location": ["tests57_66_4.py", 29, "test_matrix_of_several_numbers"], "keywords": {"test_matrix_of_several_numbers": 1, "__wrapped__": 1, "tests57_66_4.py": 1, "Folder_66": 1}, "outcome": "passed", "longrepr": null, "when": "setup", "user_properties": [], "sections": [], "duration": 0.00012524999328888953, "start": 1700336656.9758875, "stop": 1700336656.9760134, "$report_type": "TestReport", "item_index": 2, "worker_id": "gw1", "testrun_uid": "9872be2da62f4cb9b8d24115eab3360e", "node": "<WorkerController gw1>"}
{"nodeid": "tests57_66_4.py::test_matrix_of_single_number_equal_to_sum", "location": ["tests57_66_4.py", 17, "test_matrix_of_single_number_equal_to_sum"], "keywords": {"test_matrix_of_single_number_equal_to_sum": 1, "__wrapped__": 1, "tests57_66_4.py": 1, "Folder_66": 1}, "outcome": "passed", "longrepr": null, "when": "setup", "user_properties": [], "sections": [], "duration": 0.00019647699082270265, "start": 1700336656.9759562, "stop": 1700336656.9761539, "$report_type": "TestReport", "item_index": 0, "worker_id": "gw0", "testrun_uid": "9872be2da62f4cb9b8d24115eab3360e", "node": "<WorkerController gw0>"}
{"nodeid": "tests57_66_4.py::test_matrix_of_single_number_equal_to_sum", "location": ["tests57_66_4.py", 17, "test_matrix_of_single_number_equal_to_sum"], "keywords": {"test_matrix_of_single_number_equal_to_sum": 1, "__wrapped__": 1, "tests57_66_4.py": 1, "Folder_66": 1}, "outcome": "failed", "longrepr": {"reprcrash": {"path": "/usr/lib/python3/dist-packages/numpy/matrixlib/defmatrix.py", "lineno": 193, "message": "IndexError: index 1 is out of bounds for axis 1 with size 1"}, "reprtraceback": {"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    @ignore_warnings", "    def test_matrix_of_single_number_equal_to_sum():", "        input_matrix = np.matrix([[99]])", ">       assert submatrix_with_particular_sum(input_matrix) == [[[99]]]"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q57/codellama_results_4/Folder_66/tests57_66_4.py", "lineno": 21, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    temp_matrix[k, l] = matrix[i + k, j + l]"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "Q57/codellama_results_4/Folder_66/generated_answer.py", "lineno": 18, "message": "in submatrix_with_particular_sum"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    def __getitem__(self, index):", "        self._getitem = True", "    ", "        try:", ">           out = N.ndarray.__getitem__(self, index)", "E           IndexError: index 1 is out of bounds for axis 1 with size 1"], "reprfuncargs": {"args": [["self", "matrix([[99]])"], ["index", "(0, 1)"]]}, "reprlocals": null, "reprfileloc": {"path": "/usr/lib/python3/dist-packages/numpy/matrixlib/defmatrix.py", "lineno": 193, "message": "IndexError"}, "style": "long"}}], "extraline": null, "style": "long"}, "sections": [], "chain": [[{"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    @ignore_warnings", "    def test_matrix_of_single_number_equal_to_sum():", "        input_matrix = np.matrix([[99]])", ">       assert submatrix_with_particular_sum(input_matrix) == [[[99]]]"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q57/codellama_results_4/Folder_66/tests57_66_4.py", "lineno": 21, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    temp_matrix[k, l] = matrix[i + k, j + l]"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "Q57/codellama_results_4/Folder_66/generated_answer.py", "lineno": 18, "message": "in submatrix_with_particular_sum"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    def __getitem__(self, index):", "        self._getitem = True", "    ", "        try:", ">           out = N.ndarray.__getitem__(self, index)", "E           IndexError: index 1 is out of bounds for axis 1 with size 1"], "reprfuncargs": {"args": [["self", "matrix([[99]])"], ["index", "(0, 1)"]]}, "reprlocals": null, "reprfileloc": {"path": "/usr/lib/python3/dist-packages/numpy/matrixlib/defmatrix.py", "lineno": 193, "message": "IndexError"}, "style": "long"}}], "extraline": null, "style": "long"}, {"path": "/usr/lib/python3/dist-packages/numpy/matrixlib/defmatrix.py", "lineno": 193, "message": "IndexError: index 1 is out of bounds for axis 1 with size 1"}, null]]}, "when": "call", "user_properties": [], "sections": [], "duration": 0.00021052701049484313, "start": 1700336656.976426, "stop": 1700336656.9766371, "extras": [], "$report_type": "TestReport", "item_index": 0, "worker_id": "gw0", "testrun_uid": "9872be2da62f4cb9b8d24115eab3360e", "node": "<WorkerController gw0>"}
{"exitstatus": 2, "$report_type": "SessionFinish"}
